start_x: 0.0288
start_y: 0.0580
end_x: 0.025
end_y: 0.10

# Method
motion_planner: 1
visual_servoing: 1

# Visual servo Controller gain
Kp: 0.03

# Plot trajectory (Form: Pixel_coordinates = a_* Cartesian + b_)
a_x: 2587.5715
b_x: 541.10317
a_y: -3474.9482
b_y: 578.351918

# Motor values to finger angles
a_left: 0.002192
b_left: 1.027071
a_right: 0.002203
b_right: 0.325983

hold_left: 0.80
hold_right: 0.70
